### ISSUE TITLE
FIFO queue not raising an error when submitting a message to it using DelaySeconds

### ISSUE THREAD

**Issue by djones11** [getmoto/moto#7398]
When creating a SQS FIFO queue using:

```
@pytest.fixture(scope="function")
def sqs_client():
    with mock_aws():
        yield boto3.client("sqs")


@pytest.fixture(scope="function")
def test(sqs_client):
    yield sqs_client.create_queue(
        QueueName="test.fifo",
        Attributes={"FifoQueue": "true"},
    )
```
   
and then submitting a message to it using:

```
sqs_client.send_message(
        QueueUrl="test.fifo",
        MessageBody=params.model_dump_json(),
        DelaySeconds=5,
        MessageGroupId="test",
        MessageDeduplicationId=str(uuid4()),
    )
```

this should raise an error as DelaySeconds is not allowed in a fifo queue in AWS, however no error is raised when using moto.

**Comment by rafcio19** [comment]
@bblommers I can take a look at this

**Comment by bblommers** [comment]
Hi @djones11, thanks for raising this!

This should be fixed as of `moto >= 5.0.3.dev31` - please let us know if you run into any (other) issues.
